cookbooks/travis_build_environment/attributes/default.rb

```diff
@@ -252,20 +244,24 @@
 default['travis_build_environment']['neo4j_checksum'] = '24fd6a704e0d80c4b4f9a3d17ce0db23f258a8cdcfa1eb28d7803b7d1811ee96'
 
 default['travis_build_environment']['mercurial_install_type'] = 'ppa'
-default['travis_build_environment']['mercurial_version'] = '4.1.3~trusty1'
+default['travis_build_environment']['mercurial_version'] = '4.2.2~trusty1'
 
-default['travis_build_environment']['shellcheck_url'] = 'https://s3.amazonaws.com/travis-blue-public/binaries/ubuntu/14.04/x86_64/shellcheck-0.4.5.tar.bz2'
-default['travis_build_environment']['shellcheck_version'] = '0.4.5'
-default['travis_build_environment']['shellcheck_checksum'] = '82397d9993d01daee52cebb672e782c596dc7ec928a199c709a20bd24c2941c9'
+default['travis_build_environment']['shellcheck_url'] = 'https://storage.googleapis.com/shellcheck/shellcheck-v0.4.6.linux.x86_64.tar.xz'
+default['travis_build_environment']['shellcheck_version'] = '0.4.6'
+default['travis_build_environment']['shellcheck_checksum'] = 'fe0a6e94d9cf24b5a46553265846480425067f95f2630317f8fd99bc60a13719'
 default['travis_build_environment']['shellcheck_binaries'] = %w[shellcheck]
 
-default['travis_build_environment']['shfmt_url'] = 'https://github.com/mvdan/sh/releases/download/v1.0.0/shfmt_v1.0.0_linux_amd64'
-default['travis_build_environment']['shfmt_checksum'] = '172d2cf2d7d71191b7b0cd0d7169d703475825cdd270a6982899a349dcd98588'
+default['travis_build_environment']['shfmt_url'] = 'https://github.com/mvdan/sh/releases/download/v1.3.1/shfmt_v1.3.1_linux_amd64'
+default['travis_build_environment']['shfmt_checksum'] = '4cdb7b24f5de623ad5d691f826ada61b4390a1c0c7dc1ef8414a5e9c1755f7c7'
 
-default['travis_build_environment']['yarn_url'] = 'https://yarnpkg.com/downloads/0.17.8/yarn-v0.17.8.tar.gz'
-default['travis_build_environment']['yarn_version'] = '0.17.8'
-default['travis_build_environment']['yarn_checksum'] = 'b54e762e2a54f1fb23c6b0f9c239c3791aae05aface5ea0d6498f2a7979b541c'
-default['travis_build_environment']['yarn_binaries'] = %w[bin/yarn bin/yarnpkg bin/yarn.js]
+default['travis_build_environment']['yarn_url'] = 'https://yarnpkg.com/downloads/0.27.5/yarn-v0.27.5.tar.gz'
+default['travis_build_environment']['yarn_version'] = '0.27.5'
```

Comment:
When this merges I'll close out #854 :-)
(I hadn't merged myself since unsure whether you were ok with me doing so)